contracts/parent-chain/contract-upgrades/DeployEspressoOsp.s.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+import "forge-std/Script.sol";
+import "nitro-contracts.git/osp/OneStepProverMemory.sol";
```

Comment:
```suggestion
import "nitro-contracts/osp/OneStepProverMemory.sol";
```

Currently does not matche the submodule path. Following lines too.

cc @zacshowa